app/controllers/hosts_controller.rb

```diff
@@ -333,6 +333,7 @@ def forget_status
         redirect_to host_path(@host)
       end
       format.json do
+        @host.refresh_global_status!
```

Comment:
this indeed fixes the issue, but it should happen regardless of the response type, so please move it after the `status.delete`

Reply:
It does, I couldn't find the place exactly where we render it on the old page. However it's a good idea to refresh the global status anyway (even when triggered from the old UI) as clearly this resulted in the state inconsistency when we check the host through API.